src/client/opamAuxCommands.ml

```diff
@@ -498,7 +498,9 @@ let get_compatible_compiler ?repos rt dir =
   let compilers = OpamPackage.Set.inter compilers installable in
   try
     Some (OpamSolution.eq_atom_of_package
-       (OpamPackage.Set.choose_one compilers)), true
+            (OpamPackage.Set.choose_one compilers)),
+    List.exists (OpamFile.OPAM.has_flag Pkgflag_Compiler)
+      (OpamPackage.Map.values local_opams)
   with
```

Comment:
What does this bool mean ? It seems it appeared in a92c80bfb4654581427a49bd7a99a69a49bc44d4 but I can't find any documentation about it...

Reply:
cf. https://github.com/ocaml/opam/pull/4058/commits/7edd98d6678e612df76f12948d109afdfeea2087